EXAMPLE_CONFIG.json5

```diff
@@ -147,6 +147,34 @@
           /// The "memory" volume is always available, and doesn't require any per-storage options, so requesting "memory" by string is always sufficient.
           volume: "memory",
         },
+        demo2: {
+          key_expr: "demo/memory2/**",
+          volume: "memory",
+          /// If multiple storages subscribing to the same key_expr should be synchronized, declare them as replicas.
+          /// In the absence of this configuration, a normal storage is initialized
+          replica_config: {
```

Comment:
I wonder wether this type of configuration should be done per "storage", or to have a more generic section with common parameters that I expect to be rarely changed per storage-basis.

Reply:
It should ideally be per `key_expr` since that is the basic unit of replication. But some parameters should be system-wide, for example, the `align_prefix`; and I am not sure if we should let the user configure it. Maybe we don't expose it to the user until it is deemed necessary?

Reply:
I removed `align_prefix`, `subintervals`, `hot`, `warm` since it would be problematic if we let the user configure it. The first two is now hard coded into the system, and the last two are computed from other user-provided configuration.

Reply:
Moreover it is "per key_expr" rather than per storage